--- ./git-sync
+++ ./git-sync
@@ -35,13 +35,13 @@
 # succeed.
 
 # command used to auto-commit file modifications
-DEFAULT_AUTOCOMMIT_CMD="git add -u ; git commit -m \"%message\";"
+DEFAULT_AUTOCOMMIT_CMD=${DEFAULT_AUTOCOMMIT_CMD:-"git add -u ; git commit -m \"%message\";"}
 
 # command used to auto-commit all changes
-ALL_AUTOCOMMIT_CMD="git add -A ; git commit -m \"%message\";"
+ALL_AUTOCOMMIT_CMD=${ALL_AUTOCOMMIT_CMD:-"git add -A ; git commit -m \"%message\";"}
 
 # default commit message substituted into autocommit commands
-DEFAULT_AUTOCOMMIT_MSG="changes from $(uname -n) on $(date)"
+DEFAULT_AUTOCOMMIT_MSG=${DEFAULT_AUTOCOMMIT_MSG:-"changes from $(uname -n) on $(date)"}
 
 
 # AUTOCOMMIT_CMD="echo \"Please commit or stash pending changes\"; exit 1;"
